packages/interfaces/src/protocols.ts

```diff
@@ -43,6 +43,33 @@ export type ApplicationInfo = {
 
 export type ShardingParams = ShardInfo | ContentTopicInfo | ApplicationInfo;
 
+//TODO: merge this with ProtocolCreateOptions or establish distinction
```

Comment:
can we address it in this PR?

Reply:
agree, but let's add the link to the comment

```suggestion
// TODO: merge this with ProtocolCreateOptions: https://github.com/waku-org/js-waku/issues/2048
```